clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -2221,3 +2221,11 @@
       Out: ""
       MatchMode: Full
       RuleId: "remove_empty_library_arg"
+
+- Rule: rule_set_source_files_properties
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_source_files_properties_lang
+  In: set_source_files_properties(${src_files} PROPERTIES${some_other_prop}LANGUAGE CUDA${other_props})
```

Comment:
According to the define of `set_source_files_properties`:
```
set_source_files_properties(<files> ...
                            [DIRECTORY <dirs> ...]
                            [TARGET_DIRECTORY <targets> ...]
                            PROPERTIES <prop1> <value1>
                            [<prop2> <value2>] ...)
```
LANGUAGE CUDA is one value of PROPERTIES, and only works on mode "LANGUAGE CUDA", can not cover "LANGUAGE C CUDA" mode.
So I advice you add a subrule to do the operation of remove "CUDA".

Reply:
CMake only excepts one language in a single call to `set_source_files_properties`.